app/views/checkout/payment.scala.html

```diff
@@ -157,7 +158,11 @@
                                     Delivery address
                                 </legend>
                                 <div class="field-panel__fields">
-                                    @fragments.checkout.fieldsPaper(productData, countriesWithCurrency)
+                                    @if(productData.plans.default.products.seq.contains(Delivery)) {
+                                        @fragments.checkout.fieldsPaper(productData, countriesWithCurrency)
+                                    }else{
```

Comment:
The `@if(productData.planEither.isRight)` above it (line 154) should take care of that. It was already in place to switch on digital pack vs. home delivery.